### ISSUE TITLE
Allow users to specify data tables for tests

### ISSUE THREAD

**Issue by heynemann** [heynemann/pyccuracy#59]
Allow users to specify tests like this:

Given
  I go to Login Page
When
  I fill the textbox "username" with "<<username>>"
  And I fill the textbox "password" with "<<password>>"
Then
  I see the div "greeting" contains "Hello <<username>>"
With
  | username | password |
  | john          | nhoj          |
  | paul          | luap          |

This would generate two scenarios (one for john and one for paul). It would also replace <<username>> for the given username and <<password>> for the given password.

**Comment by wladston** [comment]
Would love to see this working.

**Comment by heynemann** [comment]
Also would like to see an implementation of this by someone else. Care to try?

**Comment by wladston** [comment]
Yeah, I'd love to. Also it would be my first open source contribution :)

**Comment by heynemann** [comment]
I can assure you it will be very welcome and if you'd like it can get you
committer status with pyccuracy. I have too many open source projects I
think, thus I'm not paying Pyccuracy the attention it deserves.

Cheers,
Bernardo Heynemann
Lead Dev - Pyccuracy

**Comment by wladston** [comment]
Sure! The college semester hasn't ended yet so things are on a rush now, but I'll set apart some time to create a nice path :) If you like my path, I would be happy to get the commiter status! Pyccuracy is a great project and I'm sure I can learn a lot contributing to it.
